bootstrap.ml

```diff
@@ -120,7 +120,7 @@ let ocamldep = get_prog bin_dir "ocamldep"
 let run_ocamllex src =
   let dst = String.sub src ~pos:0 ~len:(String.length src - 1) in
   let x = Sys.file_exists dst in
-  let n = exec "%s -q %s" ocamllex src in
+  let n = exec "%s -q %s" (Filename.quote ocamllex) src in
```

Comment:
You noted in https://github.com/janestreet/jbuilder/issues/322#issue-272748044 that quoting here wasn't working. Looking at the output of [this build](https://ci.appveyor.com/project/Chris00/ocaml-appveyor/build/1.0.190#L5118), the PATH in question included a forward-slash and you were also quoting `src`.

Did changing one of those two things fix the problem here? I'm happy to investigate this further, I'm just wondering if you already had?

Reply:
Sorry, I did not check all combinations as I needed to rebuild the compiler each time — my interest was to have the compiler and jbuilder ASAP to test other packages.  I'll enable more recent versions of the compiler in my fork to test it more thoroughly — I doubt I can find the time to do that before this weekend though.